src/renderer/dx/ScreenPixelShader.h

```diff
@@ -1,19 +1,46 @@
 #pragma once
 
 #ifdef __INSIDE_WINDOWS
-const char screenPixelShaderString[] = "";
+constexpr char errorPixelShaderString[] = "";

```

Comment:
TODO: _INSIDE_WINDOWS I assume is for when shipping with windows? Should we also add this ifdef to terminal effects code in DxRenderer.cpp?